plugins/docker/current_docker.ml

```diff
@@ -61,8 +61,8 @@ module Raw = struct
 
   module CCC = Current_cache.Output(Compose_cli)
 
-  let compose_cli ?(pull=true) ~docker_context ~name ~detach ~contents () =
-    CCC.set Compose_cli.{ pull } { Compose_cli.Key.name; docker_context; detach } { Compose_cli.Value.contents }
+  let compose_cli ?(pull=true) ?(up_args = []) ~docker_context ~name ~detach ~contents () =
+      CCC.set Compose_cli.{ pull } { Compose_cli.Key.name; docker_context; detach ; up_args } { Compose_cli.Value.contents }
```

Comment:
```suggestion
    CCC.set Compose_cli.{ pull } { Compose_cli.Key.name; docker_context; detach ; up_args } { Compose_cli.Value.contents }
```

Reply:
What about scoping the `CCC` module to just this function?

```ocaml
  let compose_cli ?(pull=true) ?(up_args = []) ~docker_context ~name ~detach ~contents () =
    let module CCC = Current_cache.Output(Compose_cli) in
    CCC.set Compose_cli.{ pull } { Compose_cli.Key.name; docker_context; detach ; up_args } { Compose_cli.Value.contents }
```